HISTORY.md

```diff
@@ -1,35 +1,13 @@
 ### ensmallen ?.??.?: "???"
 ###### ????-??-??
-
-### ensmallen 2.11.5: "The Poster Session Is Full"
-###### 2020-03-11
-  * Change "mathematical optimization" term to "numerical optimization" in the
-    documentation ([#177](https://github.com/mlpack/ensmallen/pull/177)).
-
-### ensmallen 2.11.4: "The Poster Session Is Full"
-###### 2020-03-03
-  * Require new HISTORY.md entry for each PR.
-    ([#171](https://github.com/mlpack/ensmallen/pull/171),
-     [#172](https://github.com/mlpack/ensmallen/pull/172),
-     [#175](https://github.com/mlpack/ensmallen/pull/175)).
-
-  * Update/fix example documentation
-    ([#174](https://github.com/mlpack/ensmallen/pull/174)).
-
-### ensmallen 2.11.3: "The Poster Session Is Full"
-###### 2020-02-19
   * Prevent spurious compiler warnings
     ([#161](https://github.com/mlpack/ensmallen/pull/161)).
-
-  * Fix minor memory leaks
-    ([#167](https://github.com/mlpack/ensmallen/pull/167)).
-
-  * Revamp CMake configuration
-    ([#152](https://github.com/mlpack/ensmallen/pull/152)).
+  
+  * ...
 
 ### ensmallen 2.11.2: "The Poster Session Is Full"
 ###### 2020-01-16
-  * Allow callback instantiation for SGD based optimizer
+  * Allow callback instantiation for SGD based optimizer 
```

Comment:
I think this was okay, without the space at the end.

Reply:
Hi, @zoq I messed up the `HISTORY.md` in the previous commit which is resolved now. Kindly have a look at it. Thanks.